Colored Sidebar Items.css

```diff
@@ -94,558 +94,105 @@ between the prefix groups are the color variables. Change or expand on any of
 these to suit your own folder structure and vault theme!
 */
 
-/* ------------------------------- 00 Prefix -------------------------------- */
-.nav-folder-title[data-path^="00"] {
-  color: var(--mint);
+/* ------------------------------- Base -------------------------------- */
+.nav-folder-title {
+  color: var(--color);
   --nav-item-color-hover: color-mix(
     in srgb,
-    var(--mint) var(--fg-contrast-amount),
+    var(--color) var(--fg-contrast-amount),
     var(--contrast-color)
   );
   --nav-item-background-hover: color-mix(
     in srgb,
-    var(--mint) var(--bg-contrast-amount),
+    var(--color) var(--bg-contrast-amount),
     transparent
   );
   --background-modifier-border-focus: color-mix(
     in srgb,
-    var(--mint) 40%,
+    var(--color) 40%,
     transparent
   );
-  --nav-collapse-icon-color: color-mix(in srgb, var(--mint) 60%, transparent);
+  --nav-collapse-icon-color: color-mix(in srgb, var(--color) 60%, transparent);
 }
-.nav-folder-title[data-path^="00"]:hover {
+.nav-folder-title:hover {
   --nav-collapse-icon-color: color-mix(
     in srgb,
-    var(--mint) 60%,
+    var(--color) 60%,
     var(--contrast-color)
   );
 }
-.tree-item-children .nav-folder:has(.nav-folder-title[data-path^="00"]) {
+.tree-item-children .nav-folder:has(.nav-folder-title) {
```

Comment:
you probably know the structure better than me, but the `:has(.nav-folder-title)` may not be necessary any longer